### ISSUE TITLE
Another missing 01-index entry

### ISSUE THREAD

**Issue by snoyberg** [haskell/hackage-server#643]
I tried to add a comment on #488, but since I'm blocked by @hvr that's not possible. Starting a new issue, CCing the assignees (@gbaz and @edsko).

There's another missing metadata file, this time for llvm-hs-5.1.1: https://github.com/snoyberg/hackage-mirror-tool/commit/afd41bebb84ac400ba8b4e7a4796c233b8c1fc18.

**Comment by cocreature** [comment]
I did get an error when I was uploading `llvm-hs-5.1.1`, I didn’t think much at the time since the package seemed to be present but seeing this I guess it could be relevant. Sadly it’s not particularly informative (at least to me):
```
> cabal upload --publish dist/llvm-hs-5.1.1.tar.gz
Uploading dist/llvm-hs-5.1.1.tar.gz...
Error uploading dist/llvm-hs-5.1.1.tar.gz: http code 500
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
"http://www.w3.org/TR/html4/strict.dtd"><html><head><title>Happstack 7.5.0.1
Internal Server Error</title></head><body><h1>Happstack
7.5.0.1</h1><p>Something went wrong here<br>Internal server
error<br>Everything has stopped</p><p>The error was "thread
killed"</p></body></html>
```

**Comment by gbaz** [comment]
A new llvm is uploaded, and the blocking issue is resolved. As such I'm going to close this in favor of #488.
